src/brain/gocdSlackFeeds/index.ts

```diff
@@ -169,6 +170,68 @@ const snsSaaSFeed = new DeployFeed({
   },
 });
 
+const discussSnSFeed = new DeployFeed({
+  feedName: 'discussEngSnSSlackFeed',
+  channelID: DISCUSS_ENG_SNS_CHANNEL_ID,
+  msgType: SlackMessage.DISCUSS_SNS_DEPLOY,
+  pipelineFilter: (pipeline) => {
+    // We only want to log the snuba pipeline
+    if (!SNS_SAAS_PIPELINE_FILTER.includes(pipeline.name)) {
+      return false;
+    }
+
+    // We only really care about creating new messages if the pipeline has
+    // failed.
+    return pipeline.stage.result.toLowerCase() === 'failed';
+  },
+  replyCallback: async (pipeline) => {
+    const [base, head] = await getBaseAndHeadCommit(pipeline);
+    const authors = head ? await getAuthors('snuba', base, head) : [];
+    // Get unique users from the authors
+    const uniqueUsers = await getUniqueUsers(authors);
+
+    // Pick at most 10 users to cc
+    const ccUsers = uniqueUsers.slice(0, 10);
+    const ccString = ccUsers
+      .map((user) => {
+        return `<@${user.slackUser}>`;
+      })
+      .join(' ');
+
+    const failedJob = pipeline.stage.jobs.find(
+      (job) => job.result.toLowerCase() === 'failed'
+    );
+    if (!failedJob) {
+      // This should never happen, but if it does, we want to know about it
+      Sentry.captureException(
+        new Error('Failed to find failed job in failed pipeline')
+      );
+      return [];
+    }
+    const gocdLogsLink = `https://deploy.getsentry.net/go/tab/build/detail/${pipeline.name}/${pipeline.counter}/${pipeline.stage.name}/${pipeline.stage.counter}/${failedJob.name}`;
+
+    const blocks = [
+      header(plaintext(`:double_vertical_bar: ${pipeline.name} has failed`)),
+      section(
+        markdown(`The deployment pipeline has failed due to detected issues in ${pipeline.stage.name}.\n
+Review the Errors*\n Review the errors in the *<${gocdLogsLink}|GoCD Logs>*.`)
```

Comment:
```suggestion
*Review the Errors*\n Review the errors in the *<${gocdLogsLink}|GoCD Logs>*.`)
```
I think you want the extra asterisk here.

Reply:
You may even want to remove the label as it looks kind of repetitive with due to there only being one item in the list rather than the 5 items in the list for getsentry.